posthog/models/integration.py

```diff
@@ -49,23 +49,28 @@ def client(self) -> WebClient:
         return WebClient(self.integration.sensitive_config["access_token"])
 
     def list_channels(self) -> List[Dict]:
-        # NOTE: Annoyingly the Slack API has no search so we have to load all channels...
+        # NOTE: We load public and private channels separately as when mixed, the Slack API pagination is buggy
+        public_channels = self._list_channels_by_kind("public_channel")
```

Comment:
we say `_by_kind` and then it is used as `type` 

so should it be `_list_channels_by_type(self, type: str)`

Reply:
and if I was being really nit-picky it could take type: `Union["public_channel","private_channel"]` instead of `str`

Reply:
That Union typing doesn't work for me. Trying to google what it should be but can't find it 😅

Reply:
Ah 🤦 `Literal` not `Union`

I had to type it into IDE to figure it out so pushed the change :)